spacy/util.py

```diff
@@ -1032,9 +1033,18 @@ def make_tempdir() -> Generator[Path, None, None]:
     YIELDS (Path): The path of the temp directory.
     """
     d = Path(tempfile.mkdtemp())
+
+    # On Windows, attempts to delete a git directory fail with permissions
+    # errors due to read only / hidden files. This mainly addresses that issue.
+    def _fix_perms(func, path, exc_info):
+        # IWUSR = give user write permission
+        os.chmod(path, stat.S_IWUSR)
```

Comment:
This should probably check first that the path is not writable, so that we can immediately re-raise the exception on other issues.

Reply:
Your previous version with `tempfile.TemporaryDirectory` worked correctly in this case, right? Or did you not use that with git at that point?

Reply:
Hmm, I had some hope that something here would work correctly in windows?

Reply:
I agree that this looks like it's going to mask other `PermissionError` cases?

Reply:
This has been removed with the latest changes, so marking this resolved.